pages/_conferenceCode/contacts/index.vue

```diff
@@ -366,18 +392,100 @@ function isFiltersApplied (_this){
   else return false
 }
 
+async function asyncFind (query){
+  this.isLoadingOrganization = true
+  if (query !== '')
+    this.organizationOptions = await this.$kronosApi.getOrganizations({
+      FreeText: query,
+      limit   : 25
+    })
+
+  this.isLoadingOrganization = false
+}
+
 async function mounted (){
+  const qs = this.$route.query
+
   this.meetingsOptions = await getMajorMinorMeetings(this)
 
-  this.organizationOptions = await this.$kronosApi.getOrganizations({
-    limit: 5000
-  })
+  // this.organizationOptions = await this.$kronosApi.getOrganizations({
+  //   limit: 25
+  // })
+
+  this.countryList = this.$store.getters['countries/asSelectOptions']
+
+  sortCountryByAsc(this.countryList, this)
+
+  this.organizationTypesOptions = this.$store.getters[
+    'organizations/asSelectOptions'
+  ]
+
+  setQueryStringFilters(this, qs)
+}
+
+function setQueryStringFilters (_this, qs){
+  if (qs.FreeText) _this.searchTagInput = qs.FreeText
+
+  if (qs.Governments)
+    _this.filter.selectedCountry = setSelectedCountry(_this, qs.Governments)
+
+  if (qs.EventUIDs)
+    _this.filter.selectedMeetings = setSelectedMeetings(_this, qs.EventUIDs)
+
+  if (qs.CountryScope)
+    _this.filter.selectedScop = setCountryScope(_this, qs.CountryScope)
+
+  if (qs.EventRegistrationStatus)
+    _this.filter.selectedAttendance = setRegistrationStatus(
+      _this,
+      qs.EventRegistrationStatus
+    )
+  if (qs.OrganizationUIDs)
+    _this.filter.selectedOrganization = setOrganizations(
+      _this,
+      qs.OrganizationUIDs
+    )
+}
```

Comment:
don't warp the parameter on multiple line for short lines. we have large screen now.
Also apply this instead of passing `this`

```suggestion
  if (qs.EventRegistrationStatus)
    this.filter.selectedAttendance = setRegistrationStatus.apply(this, qs.EventRegistrationStatus)
  if (qs.OrganizationUIDs)
    this.filter.selectedOrganization = setOrganizations.apply(this, qs.OrganizationUIDs)
}
```

Reply:
done